### PR TITLE
Fixing race condition issue with katello-agent vers with increased sleep

### PR DESCRIPTION
**Pull request by jyejare**
Continuing from PR #53 with increased common sleep for all clients.

### CODE REVIEW

File: upgrade/client.py [hunk 1]
```diff
@@ -157,9 +157,9 @@ def satellite6_client_upgrade(os_version, clients):
             clients,
             host=docker_vm
         )
-        # Fetching katello-agent version post upgrade
-        # Giving 1 minute for docker clients to upgrade katello-agent
-        time.sleep(60)
+        # Fetching katello-agent version post upgrade from all clients
+        # Giving 5 minutes for docker clients to upgrade katello-agent
+        time.sleep(300)
```
Comment (sghai) [inline upgrade/client.py]: 300 is a big sleep :/ is it really required ?

Comment (jyejare) [inline upgrade/client.py]: I believe ... For at-least eight clients 5 minutes is good i believe.

Comment (sghai) [inline upgrade/client.py]: ok.. lets give that a try and see if that fix the issue.

**Comment by sghai** [comment]
ACK

File: upgrade/client.py (hunk 1 above)
Comment (jyejare) [inline upgrade/client.py]: Yeah ! We will see the results out of it and decide the way ahead ....

Comment (san7ket) [inline upgrade/client.py]: Why not combine docker_clients_upgrade + docker_clients_katello_agent_version , we are running them separately for each hostname anyway. 1 less loop, didn't test but I guess could help with race condition 
Upgrade a container and add the version in dict

**Comment by san7ket** [comment]
ACK ,

**Pull request merged**